src/components/molecules/TablesUserList/TablesUserList.tsx

```diff
@@ -166,7 +166,7 @@ const TablesUserList: React.FunctionComponent<PropsType> = ({
         <>
           {tables.map((table: Table, i: number) => (
             <TableComponent
-              key={table.title}
+              key={table.reference}
```

Comment:
What's the intent/reasoning for this change? Is it to fix anything specifically? Or just seemed like a better choice?

Reply:
This change exactly is because it seemed to be a better choice and titles can potentially be modified thus same

Reply:
Makes sense.